src/test/java/no/difi/sdp/client2/domain/ForsendelseTest.java

```diff
@@ -14,7 +14,11 @@
 import java.util.ArrayList;
 
 import static no.difi.sdp.client2.ObjectMother.mottaker;
-import static org.fest.assertions.api.Assertions.assertThat;
+import static org.hamcrest.CoreMatchers.is;
```

Comment:
Ser også at du bruker både `is` og `equalTo`. Disse er de samme matcherne, og det er vel greiest å importere bare én av de per klasse, med mindre noe leser _vanvittig_ mye bedre med enten den ene eller den andre. Selv er jeg mest glad i `is(..)` fordi den er mest konsis.